### DISCUSSION TITLE
Final Report -

### DISCUSSION THREAD

**Discussion by raux** [NAIST-SE/AttractingContributors#508] [Final Report]
Example format: 111111 (id), https://github.com/raux/AttractingContributors-Anime, #contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/4

**Comment by shumpe-m** [comment]
2211301,https://github.com/shunpe-hash/AttractingContributors-anime, 15 contributors, [#418](https://github.com/NAIST-SE/AttractingContributors/discussions/418)

**Reply by raux** [reply]
Please list up the number of contributions

**Comment by XC1996114514** [comment]
2211331,https://github.com/XianChen1996/AttractingContributors_type_in_your_favourite_footballteam #12 contributors,#405

**Comment by AtropsCooper** [comment]
2211345, https://github.com/AtropsCooper/AttractingContributors-Random, 12 contributors, #404

**Comment by muhammadalqaaf** [comment]
2111427, https://github.com/muhammadalqaaf/AttractingContributors-WishlistCountries, #13contributors, #538

**Reply by raux** [reply]
please list up the number of contributions

**Comment by chisakondo** [comment]
2211116, https://github.com/chisakondo/favorite_spot_in_Nara,  10contributors, #502[](https://github.com/NAIST-SE/AttractingContributors/discussions/502)

**Reply by raux** [reply]
please list up the number of contributions

**Reply by chisakondo** [reply]
I'm sorry, I fixed it!

**Comment by junya-0504** [comment]
2211313, https://github.com/junya-0504/AttractingContributors_SmartphoneGame, 10 contributors, #515

**Reply by raux** [reply]
please list up the number of contributions

**Comment by kazuyo000207** [comment]
2211070, https://github.com/kazuyo000207/AttractingContributors-Curry, 11 contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/510

**Reply by raux** [reply]
please list up the number of contributions

**Comment by uriuriboo** [comment]
2211305, https://github.com/uriuriboo/AttractingContributors ,#contributors, #420

**Reply by raux** [reply]
please list up the number of contributions

**Comment by ploychanok** [comment]
2111414, https://github.com/ploychanok/AttractingContributors-WithinThreeWords, #11 contributors, #419

**Comment by gojiteji** [comment]
2211176, https://github.com/gojiteji/AttractingContributors-Trip, 11, https://github.com/NAIST-SE/AttractingContributors/discussions/406

**Comment by swindlemek** [comment]
2111430, https://github.com/swindlemek/FavoriteSport, 10 Contributors, #421

**Comment by ghost** [comment]
2211084, https://github.com/ozaki-shota/AttractingContributors_subscription_service, 10 contributors , #422

**Comment by HK-TO** [comment]
2211073, https://github.com/HK-TO/AttractingContributors-Cooking, 10 contributors, #455

**Comment by IppeiOTAKE** [comment]
2211065, https://github.com/IppeiOTAKE/AttractingContributors-OnlineGame-or-SocialGame, 12 contributors, #452

**Comment by okawaryusei** [comment]
2211060, https://github.com/okawaryusei/AttractingContributors-RPG, 12 contributors, #457

**Comment by takashi-r** [comment]
2211326, https://github.com/takashi-r/AttractingContributors-game, 14contributors, #459

**Reply by raux** [reply]
you are missing your discussion thread

**Comment by aru1702** [comment]
2111426, https://github.com/aru1702/AttractingContributors-TouhouFumo, **15 contributors**, https://github.com/NAIST-SE/AttractingContributors/discussions/407

**Comment by TaiseiURAKAMI** [comment]
2211051, https://github.com/TaiseiURAKAMI/AttractingContributors, 10 contributors, #470

**Comment by RyukiYoshida** [comment]
2211320, https://github.com/RyukiYoshida/AttractingContributors-sushi, 15 contributors, #417

**Comment by hamza-bashir01** [comment]
2111411, https://github.com/hamza-bashir01/watching-movies, 11 contributors, [[#101](https://github.com/hamza-bashir01/watching-movies/discussions/1)](url)

**Comment by ak-naist** [comment]
2211002,https://github.com/ak-naist/AttractingContributors-chain_restaurant_list/tree/aokikatsuki, #14contributors, #438

**Comment by jomi-99810** [comment]
2211246,https://github.com/jomi-99810/AttractingContributors_favoriteMovie, 17 contributors,#448

**Comment by n-takuma** [comment]
2211200, https://github.com/n-takuma/AttractingContributors-ChiledName, 19 contributors, #435

**Comment by sz-armin** [comment]
2111432, https://github.com/sz-armin/AttractingContributors-FavWord, 12 contributors, #529

**Comment by fanyoumei** [comment]
2111416, https://github.com/fanyoumei/iceCreamFlavor-, 12 contributors, #555

**Comment by s1534** [comment]
2211278, https://github.com/s1534/AttractingContributors-MusicArtist, 10 contributors, #441

**Comment by paaaaay5** [comment]
2211113,https://github.com/paaaaay5/Favorite-McDonalds-Menu, 10 contributors, #482

**Comment by SInadumi** [comment]
2211029, https://github.com/SInadumi/AttractingContributors-Musician, 15 contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/437

**Comment by vincentmichael089** [comment]
2111434, https://github.com/vincentmichael089/AttractingContributors-Kanji, 17 contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/486

**Comment by shintani-y** [comment]
2211152, https://github.com/shintani-y/AttractingContributors-Average_sleep_time_list, 18 contributors, #439

**Comment by tmymzkr** [comment]
2111409,https://github.com/m-tomoya/AttractingContributors-movie,0 contributors 5,  https://github.com/NAIST-SE/AttractingContributors/discussions/499

**Comment by sing0421** [comment]
2211177, https://github.com/sing0421/AttractingContributors-Music, 10 contributors, #467

**Comment by yoshinobu-fukumitsu** [comment]
2211255, https://github.com/yoshinobu-fukumitsu/AttractingContributors-Mouse, 16 contributors, #560

**Comment by YuseiNaito** [comment]
2211197, https://github.com/YuseiNaito/AttractingContributors-Japanese-Movie, 10 contributors, #425

**Comment by n-tsubasa-n** [comment]
2211210, https://github.com/n-tsubasa-n/AttractingContributors-RamenName, 14 contributors, #567

**Comment by haru911-99** [comment]
2211188, https://github.com/haru911-99/AttractingContributors, 10 contributors, #568

**Comment by amota-mt** [comment]
2211284, https://github.com/amota-mt/AttractingContributors-Coffee, 11 contributors, #465

**Comment by Kotjik** [comment]
2094023, https://github.com/Kotjik/AttractingContributors-FavoriteCharacter, 15 contributors, #415

**Comment by rktymt** [comment]
2211117, https://github.com/rktymt/Swimming_Style-AttractingContributors-, 10 contributors, #474

**Reply by rktymt** [reply]
Is this OK ?

**Comment by Akari000** [comment]
2211233, https://github.com/Akari000/AttractingContributors-favorite-song, 15 contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/476

**Comment by ChanYu1224** [comment]
2211224, https://github.com/ChanYu1224/AttractingContributors-MusicArtists, 25 contributors, #408

**Comment by a0163072** [comment]
2211194, https://github.com/a0163072/tell-me-your-favorite-fluits, 13contributors, #508

**Comment by keurou** [comment]
2111039, https://github.com/keurou/AttractingContributors_horse, 10contributors, #566

**Comment by KazukiSenda** [comment]
2211164, https://github.com/KazukiSenda/AttractingContributors-DreamJob, 9contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/443

**Comment by AkihiroKusuda** [comment]
2211107, https://github.com/AkihiroKusuda/AttractingContributors-VideoGame, 0contributeors, #571

**Comment by goat1010** [comment]
2211163, https://github.com/goat1010/AttractingContributors-animal, 14 contributors, #527

**Comment by mueharac** [comment]
2211048, https://github.com/tabineko/favoriteanimals, 12 contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/475

**Comment by ShuSakamoto** [comment]
2211351,https://github.com/ShuSakamoto/AttractingContributers-DevilFruit, 12 contributors,#424

**Comment by suika10000** [comment]
2111025, https://github.com/suika10000/AttractingContributors-GPU, 12 contributors, #487

**Comment by zhang9711** [comment]
2111442, https://github.com/zhang9711/AttractingContributors_City-Recommendations, 10 contributors, #539

**Comment by MGMCN** [comment]
2211332, https://github.com/MGMCN/AttractingContributors_jp_anime, 5 contributors, #403

**Comment by Yusuke196** [comment]
2111402, https://github.com/Yusuke196/AttractingContributors-Proverb, 11 contributors, #445

**Comment by sh1n00** [comment]
2211143, https://github.com/namekosiru/FavoriteSoftwareService, 14 contributors, #484

**Comment by wyx19981124** [comment]
2211341, https://github.com/wyx19981124/AttractingContributors-app, 11 contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/447

**Comment by JVasselli** [comment]
2111433, https://github.com/JVasselli/AttractingContributors, 10 contributors, #528

**Comment by Hidenori333** [comment]
2111406, https://github.com/Hidenori333/AttractingContributors-favorite-movies, 12 contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/480

**Comment by mana-makinae** [comment]
2211267, https://github.com/mana-makinae/AttractingContributors-favorite-netflix-show, 10 contributors, #488

**Comment by jun318** [comment]
2211249, https://github.com/jun318/AttractingContributors-favorite_celebrity, 14 contributors, #498

**Comment by ghost** [comment]
2211308, https://github.com/techYamasaki/AttractingContributors-emoji, 11 contributors, #434

**Comment by ghost** [comment]
2211133, https://github.com/MakotoSato0321/AttractingContributors-Anime, 8 contributors, #558

**Comment by selege** [comment]
2111440, https://github.com/selege/Good-songs, 15 contributors, #545

**Comment by daytaotao** [comment]
2111438,https://github.com/daytaotao/AttractingContributors-Chinesestar,  10 contributors,#496

**Comment by yuefufufu** [comment]
2211258,https://github.com/yuefufufu/AttractingContributors-your-favorite-MS-of-Gundam, 12 contributors,#512

**Comment by OguniShun** [comment]
2211080,https://github.com/OguniShun/AttractingContributors_Table-Game, 8contributors,#542

**Comment by t-yujiro** [comment]
2211186,https://github.com/t-yujiro/AttractingContributors-Tourism, 14contributors,#10

**Comment by WeiqiZhang1999** [comment]
2211343, https://github.com/WeiqiZhang1999/AttractingContributors_Games, 10 contributors, #409

**Comment by teilwal-dot** [comment]
2211329, https://github.com/teilwal-dot/AttractingContributors-Manga, 14 contributors, #489

**Comment by KokiOkai** [comment]
2211075, https://github.com/KokiOkai/AttractingContributors_Music, 11 contributors, #541

**Comment by jude509** [comment]
2211359, https://github.com/jude509/AttractingContributors-favorite-music, 11 contributors, [#526](https://github.com/NAIST-SE/AttractingContributors/discussions/526)

**Comment by mlieynua** [comment]
2211072, https://github.com/mlieynua/AttractingContributors-favorite-movie, 12 contributors, #479

**Comment by yoriokahiroto** [comment]
2211324, https://github.com/yoriokahiroto/YourBirthPlace,16 contributors, #469

**Comment by ryuta-messi** [comment]
2211018, https://github.com/ryuta-messi/AttractingContributors, 13 contributors,#451

**Comment by mmm-nana** [comment]
2211281, https://github.com/mmm-nana/AttractingContributors-hobby, 11 contributors, #473

**Comment by ShizukuIida** [comment]
2211012, https://github.com/ShizukuIida/AttractingContributors-Food, 10 contributors, #534

**Comment by ninkofu** [comment]
2211221, https://github.com/7518085/AttractingContributors-Please-tell-me-your-favorite-Electronics-company, 11 contributors, #468

**Comment by kazutomo2019** [comment]
2211355, https://github.com/kazutomo2019/Recently-cooked-food/blob/main/README.md, 15 contributors, #532

**Comment by hatodove22** [comment]
2211063, https://github.com/hatodove22/Your-Phone, 15 contributors, #517

**Comment by Tomohiro-git** [comment]
2111191, https://github.com/Tomohiro-git/AttractingContributors_impressive_place, 10 contributors, [#531](https://github.com/NAIST-SE/AttractingContributors/discussions/531)

**Comment by Y-eng** [comment]
2211311, https://github.com/Y-eng/AttractingContributors-writer, 11 contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/491

**Comment by natsuonorepositories** [comment]
2211078, https://github.com/NAIST-SE/AttractingContributors/discussions, 13 contributors, #509

**Comment by h1karu-s** [comment]
2211141, https://github.com/hikaru-sh/AttractingContributors-Ramen, 13 contributors, #519

**Comment by masaki12-s** [comment]
2211135, https://github.com/masaki12-s/AttractingContributors-StreamerList, 25 contributors, #411

**Comment by gabriel6181997** [comment]
2211360, https://github.com/gabriel6181997/AttractingContributors---TVDrama, 8 contributors, #546

**Comment by yuya-isaka** [comment]
2111401, https://github.com/yuya-isaka/AttractingContributors-ISA, 1 contributor, #458

**Comment by yuito1227** [comment]
2211010, https://github.com/yuito1227/tell-me-your-homecountry, 20 contributors, #535

**Comment by Hikaru7777** [comment]
2211248, https://github.com/Hikaru7777/your-favorite-color.git , 12 contributors , #565

**Comment by gilugamesh0711** [comment]
2111206, https://github.com/gilugamesh0711/AttractingContributors_your_favorite_heroine, 5 contributors, #533

**Comment by yushi-okunishi** [comment]
2211081, https://github.com/yushi-okunishi/AttractingContributors-Games, 12 contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/536

**Comment by k0tk267** [comment]
2211067, https://github.com/k0tk267/AttractingContributors, 10 contributers, #492

**Comment by dodo3095** [comment]
2111013, https://github.com/dodo3095/AttractingContributors-LocalFoods, 10 contributors, #550

**Comment by naotos575** [comment]
2211151, https://github.com/naotos575/AttractingContributors-LUNCH.git, 12contributors, #520

**Comment by Lee-AHCL** [comment]
2111421, https://github.com/Lee-AHCL/AttractingContributors-Foreign_Language, 12 Contributors, #537

**Comment by toma-ko** [comment]
2211114, https://github.com/toma-ko/Social_Developer#homework6---which-do-you-like-better-rice-or-bread

**Comment by ani-hovhannisyan** [comment]
2111418 https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git 13 contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998

**Comment by throbotaith** [comment]
2211192 https://github.com/throbotaith/AttractingContributors-Years 10 contributors,[#444](https://github.com/NAIST-SE/AttractingContributors/discussions/444l)

**Comment by selege** [comment]
2111440,https://github.com/selege/Good-songs, 15 contributors, #545

**Comment by 2220IKE** [comment]
2211017, https://github.com/2220IKE/AttractingContributors-anime, 8contributors, #562

**Comment by NakataCrimson39** [comment]
2211203, https://github.com/NakataCrimson39/Tell_me_what_music_you_like-- , 8contributors, #563

**Comment by seiji-shimizu** [comment]
2211147, https://github.com/seiji-shimizu/AttractingContributors, 10 contributors, [#549](https://github.com/NAIST-SE/AttractingContributors/discussions/549)

**Comment by YutaHmahc** [comment]
2211241,https://github.com/YutaHmahc/AttractingContributors_LastGame, 18 contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/440

**Comment by N-Eiki** [comment]
2211201, https://github.com/N-Eiki/AttractingContributors-favorite-food, 11 contributors, #453

**Comment by Yuki-Ohishi** [comment]
2211057, https://github.com/Yuki-Ohishi/What-kind-of-tea-do-you-like-.git, 11 contributors, #561

**Comment by yyshuo** [comment]
2111441，https://github.com/yyshuo/AttractingContributors-city 13 contributors,https://github.com/NAIST-SE/AttractingContributors/discussions/570

**Comment by watagit** [comment]
2211087, https://github.com/watagit/AttractingContributors-Hobby, 10 contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/578

**Comment by lehend1010** [comment]
2111423, https://github.com/Lehend1010/AttractingContributors, 14 contributors, #525

**Comment by yamaokayuki20** [comment]
2211306, https://github.com/yamaokayuki20/YourFavoriteSoftwareTool, 6 contributors, #564

**Comment by Bison779** [comment]
2211025,https://github.com/Bison779/AttractingContributors-favorite_robot,8 contributors,#504

**Comment by pui1** [comment]
2211032, https://github.com/pui1/AttractingContributors-Game, 7contributors, https://github.com/NAIST-SE/AttractingContributors/discussions/557</div>

**Comment by LBvai** [comment]
2111417, https://github.com/RuixunH/AttractingContributors-Music-from-film, 11 contributors, #518

**Comment by ghost** [comment]
2211230,https://github.com/takadakenshi/AttractingContributors, 13 contributors,https://github.com/NAIST-SE/AttractingContributors/discussions/481

**Comment by RyotoMinami** [comment]
2211283,https://github.com/RyotoMinami/AttractingContributors_Number_of_Internship.git, 10 contributors, #540

**Comment by ishizuka-yuk** [comment]
2211019,https://github.com/yukidaruma4869/FavoriteVideoGames.git,15 contributors, #500

**Comment by renbowroad** [comment]
2211212, https://github.com/renbowroad/YourFavoriteWebService 6 contributors ,#569
